javascript/example_code/s3/create-presignedURL/ReadMe.md

```diff
@@ -0,0 +1,60 @@
+# README for Generating Pre-signed URLs
+
+## Purpose
+
+This example uses a Node.js function to generate a pre-signed URL that uploads a specified file to a specified S3 bucket. A pre-signed URL grants temporary access to users who don’t have permission to directly run AWS operations in your account. 
+
+## Prerequisites
+
+To build and run this example, you need the following:
+- Node.js. For more information about installing Node.js, see the [Node.js website](https://nodejs.org).
+- The AWS SDK for JavaScript.  For more information about installing the AWS SDK for JavaScript, see see *Step 3* on the [Getting Started in Node.js in the JavaScript SDK Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/getting-started-nodejs.html).
+- AWS credentials, either configured in a local AWS credentials file, or by setting the AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables. For more information, see  [Setting Credentials in the JavaScript SDK Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/setting-credentials.html).
+- Create an S3 bucket. For more information, see [Working with Amazon S3 Buckets](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html#create-bucket-intro).
+
+## Assumptions
+Anyone with valid security credentials can create a pre-signed URL. However, the pre-signed URL must be created by someone who has permission to perform the operation that the pre-signed URL is based on.
+
+## Running the code
+1. Create a local folder.
+2. In the command line navigate to the folder you created and enter the following command (in command line) to create a package.json:
```

Comment:
**At a** command **prompt,** navigate ... enter the following command ~~(in command line)~~ to...